dev/tools/controllerbuilder/pkg/codebot/tool_run_cmd.go

```diff
@@ -65,6 +65,8 @@ func (t *RunTerminalCommand) Run(ctx context.Context, c *Chat, args map[string]a
 		cmd = exec.CommandContext(ctx, "make", tokens[1:]...)
 	case "gcloud":
 		cmd = exec.CommandContext(ctx, "gcloud", tokens[1:]...)
+	case "controllerbuilder":
```

Comment:
Very meta. Would be good to consider how we prevent infinite recursion. Can we add a tracking flag for depth and prevent depth greater than something fairly shallow (2?) until we understand it?